module/Application/src/View/Model/ApplicationOverviewSection.php

```diff
@@ -25,17 +25,17 @@ public function __construct($ref, $data, $sectionDetails)
         switch ($status) {
             case 1:
                 $mode = 'edit';
-                $statusText = 'INCOMPLETE';
+                $statusText = 'Incomplete';
                 $statusColour = 'orange';
                 break;
             case 2:
                 $mode = 'edit';
-                $statusText = 'COMPLETE';
+                $statusText = 'Complete';
                 $statusColour = 'green';
                 break;
             default:
                 $mode = 'add';
-                $statusText = 'NOT STARTED';
+                $statusText = 'Not Started';
```

Comment:
Better as `Not started`? To be consistent casing and follow the GDS guidelines.

Reply:
Yes good shout.